README.md

```diff
@@ -103,6 +103,14 @@ go install github.com/sky-uk/vergo@latest
 
   `vergo get current-version --tag-prefix=banana`
 
+* returns the current tag/release prefixed with banana, maybe a SNAPSHOT, using the first tag matched in the commit history 
+
+  `vergo get current-version --tag-prefix=banana --enable-first-tag-match`
```

Comment:
This use case makes sense. Yet I think we need to establish a language.

The way I can see this change being differentiated is the following terms:
- latest release - this would base the current version off of the latest semantic tag. (not latest based on push time etc). This would be the default)
- nearest release - this would base the current version off the nearest semantic tag. This would be overriden.

Therefore I would likely think the most clear phrasing could be:

`--nearest-release`

What are your thoughts on this?

Reply:
That is definitely a better name.

Going to make the change ASAP